mock-core-configs/etc/mock/templates/epel-10.tpl

```diff
@@ -2,10 +2,35 @@ config_opts['chroot_setup_cmd'] += " epel-rpm-macros"
 
 config_opts['dnf.conf'] += """
 
+# The metalinks below differ from the ones used in epel-release.  The
+# epel-release metalinks rely on a dnf variable expansion feature that is only
+# available since dnf 4.18.0.  In some configurations mock will be using the
+# host dnf to bootstrap the chroot, which could be older and lack those
+# features, leading to problems.  The alternative is to use this simplified
+# metalink here in this template for CentOS Stream, and a different metalink in
+# another template for RHEL.
+
+[epel]
+name=Extra Packages for Enterprise Linux $releasever - $basearch
+metalink=https://mirrors.fedoraproject.org/metalink?repo=epel-$releasever_major&arch=$basearch
+gpgkey=file:///usr/share/distribution-gpg-keys/epel/RPM-GPG-KEY-EPEL-$releasever_major
```

Comment:
Was `$releasever_major` backported to RHEL 8 DNF? Otherwise, this might only work on RHEL 9+ unless you use a bootstrap image.

Reply:
This was discussed at length in #1427, and is why this PR also defines `releasever_major` in centos-stream-10.tpl.

Reply:
The problem Neal describes (at least IIUC) is that Mock doesn't map its configuration options to DNF variables. Would you mind doing `repo=epel-{{ releasever_major }}` instead?